src/controls/animatedStatusBar.ts

```diff
@@ -1,6 +1,7 @@
-/*---------------------------------------------------------
- * Copyright (C) Microsoft Corporation. All rights reserved.
- *--------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation.
+ *  Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```

Reply:
Please fix in other TS files

Reply:
I think in *.ts files block comments headers are more conventional. Also, I wanted to be consistant with microsoft/vscode, see <https://github.com/microsoft/vscode/issues/96747>.

Reply:
This is a required style.